widget/widgetAssets/locales/en.json

```diff
@@ -22,6 +22,8 @@
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_DROP_A_FILE":"Drop a file",
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_USE_STANDARD_UPLOADER":"or click to use standard uploader",
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_FILE_SIZE_SHOULD_NOT_EXCEED_25MB":"File Size should not exceed 25MB",
+        "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_FILE_CONTENT_INVALID": "File content is null or invalid",
```

Comment:
File content is null or invalid --> File Content is Null or Invalid

Reply:
DONE